src/optimizer/group.cpp

```diff
@@ -27,20 +29,29 @@ Group::Group(GroupID id, std::unordered_set<std::string> table_aliases)
 
 void Group::AddExpression(std::shared_ptr<GroupExpression> expr,
                           bool enforced) {
+
+  // Additional assertion checks for AddExpression() with AST rewriting
+  // TODO(ncx): get group expression type
+  // if (std::is_same<Node, AbsExpr_Container>::value) {
```

Comment:
Is there a reason these are currently commented out?

Reply:
Resolving.